structure/hash.go

```diff
@@ -127,6 +134,10 @@ func (t *TxStructure) updateHash(key []byte, field []byte, fn func(oldValue []by
 		return errors.Trace(err)
 	}
 
+	if t.ignoreHashMeta {
```

Comment:
When TiDB downgrades to an older version that reads and writes hash meta, but now the hash meta is all wrong, what will happen?

Reply:
It doesn't matter, even in the downgrade case the data is wrong, old TiDB never use those data.

Reply:
I don't think so. For example, `HDel` checks meta hash, and if `meta.IsEmpty()`, it will report an error.

Reply:
So we can highlight the downgrade compatibility? there are many new feature, if TiDB upgrade and use the new feature, then downgrade, there will be a problem. Any suggestion?